Fix translation: themes/using-a-gatsby-theme (#258)
gatsbyjs/gatsby-pt-BR: docs/docs/themes/using-a-gatsby-theme.md

title: Usando um tema Gatsby
---

> Além de você [iniciar rapidamente um novo projeto utilizando o starter de tema do Gatsby](/docs/themes/getting-started/), você também pode instalar um tema Gatsby diretamente em um site Gatsby já existente. Os temas do Gatsby são plugins, então você pode [instalar e usar como qualquer outro plugin do Gatsby](/docs/using-a-plugin-in-your-site/).

Reviewer: Os links para os arquivos tão quebrado, atentar um pouco nisso.

```suggestion
Além de você [iniciar rapidamente um novo projeto utilizando o starter de tema do Gatsby](docs/docs/themes/getting-started.md), você também pode instalar um tema Gatsby diretamente em um site Gatsby já existente. Os temas do Gatsby são plugins, então você pode [instalar e usar como qualquer outro plugin do Gatsby](docs/docs/using-a-plugin-in-your-site.md).
```

## Instalando um Tema

> Como qualquer plugin do Gatsby, os temas do Gatsby são pacotes Node.js, então você pode instalá-los como qualquer outro pacote publicado em Node usando npm ou [yarn, incluindo workspaces locais](#using-yarn-workspaces).

Reviewer: ```suggestion
Como qualquer outro plugin do Gatsby, os temas do Gatsby são pacotes do Node.js. Então você pode instalá-los utilizando algum gerenciador de pacotes do Node.js como o `npm` ou o [yarn, incluindo workspaces locais](#using-yarn-workspaces).
```

      resolve: `gatsby-theme-blog`,
      options: {
        /*

>       - basePath defaults to `/`

Reviewer 2: Traduz os comentários também por favor :)

Reviewer 2: Falta essa parte aqui @lancaluis, mas não sei é realmente necessário, o que vocês acham @thayannevls, @iaurg e @jessescn?

Reviewer 3: Como é um comentário explicativo creio que possa ajudar na documentação a tradução.

}
```

Para aprender como personalizar ainda mais um tema, confira as documentações em [Gatsby theme shadowing](/docs/themes/shadowing/).

## Publicando temas

> Temas públicos do Gatsby são publicados no npm para todos utilizá-lo. Você também pode publicar um tema privado para usar na sua empresa. Exemplos de hospedagem de temas privados incluem o [npm registry](https://docs.npmjs.com/about-private-packages) e o [GitHub Package Registry](https://help.github.com/en/github/managing-packages-with-github-package-registry/about-github-package-registry).

Reviewer 2: ```suggestion
Temas públicos do Gatsby são publicados no npm para qualquer um usar. Você também pode publicar um tema privado para usar na sua empresa. Exemplos de hospedagem de temas privados incluem o [npm registry](https://docs.npmjs.com/about-private-packages) e o [GitHub Package Registry](https://help.github.com/en/github/managing-packages-with-github-package-registry/about-github-package-registry).
```

> ## Usando Yarn Workspaces

Reviewer: ```suggestion
## Usando Workspaces Yarn
```

> Se você gostaria de trabalhar com temas não publicados, considere [configurar o Yarn Workspaces para o desenvolvimento de temas](/blog/2019-05-22-setting-up-yarn-workspaces-for-theme-development/) e [usando yarn](/docs/gatsby-cli/#how-to-change-your-default-package-manager-for-your-next-project) ao invés do npm.

Reviewer: ```suggestion
Se você gostaria de trabalhar com temas não publicados, considere [configurar o Workspaces Yarn para o desenvolvimento dos temas](/blog/2019-05-22-setting-up-yarn-workspaces-for-theme-development/) e [usar o yarn](/docs/gatsby-cli/#how-to-change-your-default-package-manager-for-your-next-project) ao invés do npm.
```
